src/components/MenuFlyout/index.tsx

```diff
@@ -80,23 +82,19 @@ const MenuFlyout = (props: {
   hasArrow?: boolean
 }) => {
   useOnClickOutside(props.node, props.isOpen && !isMobile ? props.toggle : undefined)
-  const BrowserStyle = useMemo(
```

Comment:
I think we should keep useMemo. This PR shouldn't update logics

Reply:
this violate rule of hook

Reply:
This is probably because the SC is used dynamically (inside another component)